talisman/talisman.py

```diff
@@ -101,12 +109,21 @@ def init_app(
         self.frame_options_allow_from = frame_options_allow_from
 
         self.strict_transport_security = strict_transport_security
-        self.strict_transport_security_max_age =\
+        self.strict_transport_security_max_age = \
             strict_transport_security_max_age
-        self.strict_transport_security_include_subdomains =\
+        self.strict_transport_security_include_subdomains = \
             strict_transport_security_include_subdomains
 
         self.content_security_policy = content_security_policy.copy()
+        self.content_security_policy_report_uri = \
+            content_security_policy_report_uri
+        self.content_security_policy_report_only = \
+            content_security_policy_report_only
+        if self.content_security_policy_report_only and \
+                self.content_security_policy_report_uri is None:
+            raise ValueError(
+                'The parameter content_security_policy_report_only '
+                'reqires setting content_security_policy_report_uri, too.')
```

Comment:
s/reqires/requires, and you can do away with the comma. Or you can word it as `Setting content_security_policy_report_only to True also requires a URI to be specified in content_security_policy_report_uri`.